massa-models/src/settings.rs

```diff
@@ -41,3 +43,42 @@ pub struct CompactConfig {
     pub block_reward: Amount,
     pub roll_price: Amount,
 }
+
+impl Display for CompactConfig {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        writeln!(
+            f,
+            "    Genesis timestamp: {}",
+            self.genesis_timestamp.to_utc_string()
+        )?;
+        if let Some(end) = self.end_timestamp {
+            writeln!(f, "    End timestamp: {}", end.to_utc_string())?;
+        }
+        writeln!(f, "    Thread count: {}", self.thread_count)?;
+        writeln!(f, "    t0: {}", self.t0)?;
+        writeln!(f, "    delta_f0: {}", self.delta_f0)?;
+        writeln!(
+            f,
+            "    Operation validity periods: {}",
+            self.operation_validity_periods
+        )?;
+        writeln!(f, "    Periods per cycle: {}", self.periods_per_cycle)?;
+
```

Comment:
Why all these spaces?

Reply:
For the get_status command to look nice in the interactive client.

It's the workaround we're using everywhere else :/

Reply:
I'm talking about empty lines in code, but never mind...

Reply:
Ow I can fix it :)